### PR TITLE
Add SQL queries for MySQL example

### PR DESCRIPTION
**Pull request by victorcolombo**

### CODE REVIEW

File: blog-mysql-seed/mysql-seeder/seed.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO User (UserID, FirstName) VALUES (1, "Peter");
+INSERT INTO User (UserID, FirstName) VALUES (2, "Jorge");
+INSERT INTO Post (PostID, Content, AuthorUserID) VALUES (0, "Lorem ipsum dolor sit amet", 2);
```
Comment (gbouv) [inline blog-mysql-seed/mysql-seeder/seed.sql:3]: nit: newline

File: blog-mysql-seed/mysql-seeder/seed.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO User (UserID, FirstName) VALUES (1, "Peter");
```
Comment (gbouv) [inline blog-mysql-seed/mysql-seeder/seed.sql:1]: Not a SQL pro, but capital letters in column names looks weird to me at first glance. I'm more used to snake case for SQL in general, not sure if that's the standard or not. IIRC SQL is not case sensitive, so snake case is maybe better here?

File: blog-mysql-seed/mysql-seeder/setup.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    FirstName VARCHAR(255) NOT NULL,
+    PRIMARY KEY (UserID)
+);
+CREATE TABLE Post (
+    PostID INT NOT NULL AUTO_INCREMENT,
+    Content VARCHAR(255) NOT NULL,
+    AuthorUserID INT NOT NULL,
+    PRIMARY KEY (PostID),
+    FOREIGN KEY (AuthorUserID) REFERENCES User(UserID)
+);
```
Comment (gbouv) [inline blog-mysql-seed/mysql-seeder/setup.sql:12]: nit: newline

**Review by gbouv — APPROVED** [review]
Approving to unblock but without being familiar with SQL, the Camel case in SQL I think is a bit of an anti pattern, Could be wrong though

File: blog-mysql-seed/mysql-seeder/setup.sql (hunk 3 above)
Comment (victorcolombo) [reply blog-mysql-seed/mysql-seeder/setup.sql]: Done

File: blog-mysql-seed/mysql-seeder/seed.sql (hunk 1 above)
Comment (victorcolombo) [reply blog-mysql-seed/mysql-seeder/seed.sql]: Done

**Pull request merged**